src/app/core/services/location.service.ts

```
@@ -44,6 +39,18 @@ export class LocationService {
     );
   }
 
+  @Cacheable()
+  get<T>(url: string, config = {}): Observable<T> {
+    return this.httpClient.get<T>(this.ROOT_ENDPOINT + '/location' + url, config);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)